intake/catalog/tests/test_remote_integration.py

```diff
@@ -290,3 +290,18 @@ def test_getitem_and_getattr(intake_server):
     assert catalog.arr is catalog['arr']
     assert isinstance(catalog.arr, RemoteCatalogEntry)
     assert isinstance(catalog.metadata, (dict, type(None)))
+
+
+def test_search(intake_server):
+    catalog = Catalog(intake_server)
+
+    results = catalog.search('text', depth=1)
```

Comment:
I may have missed it: is this doing the normal walk/text search, or is everything passed to the server now?

Reply:
Because `catalog` is a `RemoteCatalog`, everything is passed to the server now. Previously -- that is, on current `master`, `RemoteCatalog` does not implement the `search` method at all. The `search` behavior of a local Catalog is unchanged.